### DISCUSSION TITLE
Proxmox enhanced stats unavailable when single node offline

### DISCUSSION THREAD

**Discussion by Brandoskey** [linuxserver/Heimdall#1000] [App issues]
I have a 2 node cluster with a qdevice and with one node down I lose enhanced stats. I've confirmed the IP address is pointing to the node that is still up and hitting the test button declares a successful connection to the api.

I've also tried limiting stats to the online node with no joy.

**Comment by aseemann** [comment]
Same problem here and I think I found the reason. I realy like to provide a patch but could not find the corresponding repository or code in the heimdall repo. I'm also unable to open an issue on https://apps.heimdall.site/applications/enhanced.
